### PR TITLE
Add option to edit a single message instead of sending new ones

### PR DESCRIPTION
**Pull request by chjudge**
I forked this because my friend wanted to know who was online in our server. I added functionality to allow the API to edit a single message instead of sending a new message every time someone leaves or joins. This reduces notification spam and allows the users to pin a message that can be read at a glance. I put the feature behind a flag in the config, just now thought that it might be nice to have an option for both so that you could be notified and also see a list of everyone. Anyways I thought this might be a welcome contribution to your project.

### CODE REVIEW

**Comment by eone666** [comment]
it looks like it's possible to use both methods at the same time, like two different functions of the same plugin

**Comment by eone666** [comment]
yes, both are better

**Comment by eone666** [comment]
I will be grateful for such a contribution

File: .gitignore [hunk 1]
```diff
@@ -115,3 +115,4 @@ run/
 
 # Avoid ignoring Gradle wrapper jar file (.jar files are usually ignored)
 !gradle-wrapper.jar
+/.idea/
```
Comment (eone666) [inline .gitignore:118]: i keep this directory for myself but i can only store it locally

**Comment by eone666** [comment]
and I think that it’s not a bad opportunity to store additional information in a pinned message, like an ip address

**Comment by chjudge** [comment]
The commit I just added adds the option to set the method field in the config file to BOTH to allow both options

**Comment by chjudge** [comment]
> yes, both are better

The commit I just added adds the option to set the method field in the config file to 'BOTH' to allow both options

**Comment by eone666** [comment]
@chjudge 

I did it my way to support a certain architecture in the project, but thanks for the idea

https://github.com/eone666/telegram-notifier/compare/509a7f6d6626da7ceb522eefcc9d3ae25e2ab086...main

**Comment by eone666** [comment]
only now I don’t know whether to release the update as a minor(1.1.0) version or as a major one(2.0.0)?

**Comment by chjudge** [comment]
> only now I don’t know whether to release the update as a minor(1.1.0) version or as a major one(2.0.0)?

I would consider it a minor release

**Pull request closed without merging**